js/components/cohort-definition-browser.js

```diff
@@ -96,9 +96,9 @@ define([
 			return '<span class="linkish">' + d.name + '</span>';
 		}
 
-		rowClick (d) {
-			this.selected(d.id);
-		}
+		rowClick(data) {
+            this.selected(data);
+        }
```

Comment:
fix formatting

Reply:
+ why not to call an incoming parameter `id`?

Reply:
I believe the rowClick handler receives the entire row of data, so it's appropriate to specify 'data' here instead of 'id'.